desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -0,0 +1,84 @@
+            We're all passionate about the preservation of internet history, and we were drawn to working on this
+            project to help preserve the many websites and plethora of content that will no longer be accessible
+            when users can no longer run the official Flash Player.
+            If you would like to help support this project, we welcome all contributions of any kind –
+            even if it's just playing some old games and seeing how well they run.
+        </p>
+    </description>
+    <categories>
+        <category>Game</category>
+        <category>Utility</category>
+        <category>Graphics</category>
+        <category>AudioVideo</category>
+        <category>Player</category>
+        <category>Viewer</category>
+    </categories>
+    <keywords>
+        <keyword>flash</keyword>
+        <keyword>swf</keyword>
+    </keywords>
+    <url type="homepage">https://ruffle.rs</url>
+    <url type="bugtracker">https://github.com/ruffle-rs/ruffle/issues</url>
+    <url type="faq">https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users</url>
+    <url type="translate">https://crowdin.com/project/ruffle</url>
+    <url type="donation">https://opencollective.com/ruffle</url>
+    <url type="contact">https://discord.gg/ruffle</url>
+    <url type="vcs-browser">https://github.com/ruffle-rs/ruffle</url>
+    <url type="contribute">https://github.com/ruffle-rs/ruffle/blob/master/CONTRIBUTING.md</url>
+    <screenshots>
+        <!-- TODO Update URLs to point to a commit -->
+        <screenshot environment="gnome:dark" type="default">
+            <image>https://raw.githubusercontent.com/ruffle-rs/ruffle/master/desktop/packages/linux/screenshots/dark-launcher.png</image>
+        </screenshot>
+        <screenshot environment="gnome:dark">
+            <image>https://raw.githubusercontent.com/ruffle-rs/ruffle/master/desktop/packages/linux/screenshots/dark-bloonstd.png</image>
+        </screenshot>
+        <screenshot environment="gnome:dark">
+            <image>https://raw.githubusercontent.com/ruffle-rs/ruffle/master/desktop/packages/linux/screenshots/dark-learntofly.png</image>
+        </screenshot>
+        <screenshot environment="gnome">
+            <image>https://raw.githubusercontent.com/ruffle-rs/ruffle/master/desktop/packages/linux/screenshots/light-launcher.png</image>
+        </screenshot>
+    </screenshots>
+    <provides>
+        <mediatype>application/x-shockwave-flash</mediatype>
+        <mediatype>application/vnd.adobe.flash.movie</mediatype>
+    </provides>
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>gamepad</control>
+    </supports>
+    <content_rating type="oars-1.0"/>
+    <branding>
+        <color type="primary" scheme_preference="light">#4a6cb7</color>
+        <color type="primary" scheme_preference="dark">#2c4170</color>
```

Comment:
Is there need/way to add two shades of "Ruffle orange" as well?

Reply:
These colors are used as a background behind the app icon. There was a discussion on Discord which colors to choose; in general people did not like the orange background and we chose shades of blue instead.